structopt-derive/src/attrs.rs

```diff
@@ -76,6 +76,13 @@ impl Attrs {
                 self.methods = methods.into_iter().filter(|m| m.name != name).collect();
             }
             ("name", new_name) => self.name = new_name.into(),
+            ("validator", _) => {
+                use quote::ToTokens;
+                self.methods.push(Method {
+                    name: name.to_string(),
+                    args: Ident::new(arg, Span::call_site()).into_token_stream(),
```

Comment:
An identity is very llimiting. If custom attributes now support identity, it must work for any command.

Reply:
I'm not sure what you mean by "custom attributes" - can you explain more?